index.md

```diff
@@ -20,3 +20,12 @@ Users can create non-production environments and get access to the Kubernetes AP
 {% for article in getting_started_groups %}
   [{{ article.title }}]({{ article.url | relative_url }})
 {% endfor %}
+
+## Kubernetes
```

Comment:
I'd like to leave out the word "Kubernetes" as much as possible and instead think about the "Cloud Platform", which will be based around Kubernetes but have other things to. 

So I'd put this page under the "Gettings started" section (given that it is one of the first things you will do).